docs/outputs.rst

```diff
@@ -4,97 +4,151 @@ Outputs of tedana
+                                                    of echoes, starting with 2.
+*_desc-ascendingEstimates_s0.nii.gz                 Voxel-wise S0 estimates using ascending numbers
+                                                    of echoes, starting with 2.
+*_desc-full_t2s.nii.gz                              Full T2* map/time series. The difference between
+                                                    the limited and full maps is that, for voxels
+                                                    affected by dropout where only one echo contains
+                                                    good data, the full map uses the single echo's
+                                                    value while the limited map has a NaN. Only used
+                                                    for optimal combination.
+*_desc-full_s0.nii.gz                               Full S0 map/time series. Only used for optimal
+                                                    combination.
+*_desc-initialTEDICA_mixing.tsv                     Mixing matrix (component time series) from ICA
+                                                    decomposition.
+*_echo-[echo]_desc-optcomAccepted_bold.nii.gz       High-Kappa time series for echo number ``echo``
+*_echo-[echo]_desc-optcomRejectedMidK_bold.nii.gz   Mid-Kappa time series for echo number ``echo``
+*_echo-[echo]_desc-optcomRejected_bold.nii.gz       Low-Kappa time series for echo number ``echo``
+*_echo-[echo]_desc-optcomDenoised_bold.nii.gz       Denoised time series for echo number ``echo``
+=================================================   =====================================================
 
 If ``gscontrol`` includes 'gsr':
 
-======================    =====================================================
-Filename                  Content
-======================    =====================================================
-T1gs.nii                  Spatial global signal
-glsig.1D                  Time series of global signal from optimally combined
-                          data.
-tsoc_orig.nii             Optimally combined time series with global signal
-                          retained.
-tsoc_nogs.nii             Optimally combined time series with global signal
-                          removed.
-======================    =====================================================
+=========================================   =====================================================
+Filename                                    Content
+=========================================   =====================================================
+*_desc-T1gs_bold.nii.gz                     Spatial global signal
+*_globalSignal_regressors.tsv               Time series of global signal from optimally combined
+                                            data.
+*_desc-optcomWithGlobalSignal_bold.nii.gz   Optimally combined time series with global signal
```

Comment:
Isn't this already covered on L22 ?

Reply:
The `gsr` method outputs two files that contain the same contents, I guess, although I think the duplicate is actually `*_desc-optcomNoGlobalSignal_bold.nii.gz`.